### PR TITLE
ARM64 support for docker images

### PR DESCRIPTION
**Pull request by FarisZR**
### Changes
This PR adds ARM64 builds for plausible docker images.
I tested the builds, and everything worked.

Test builds:
https://github.com/FZR-forks/analytics/pkgs/container/plausible

the latest tag is the self-hosting version
the master version is built to the latest commit

**This PR requires switching from yandex/clickhouse-server to a recent [clickhouse/clickhouse-server](https://hub.docker.com/r/clickhouse/clickhouse-server) image which supports ARM64**

I tested it using the latest version, which has an ARM64 image.

If this PR gets merged, I can create another one to update the hosting repo.

Related:
#1286 
#903

### Tests
- [ ] Automated tests have been added
- [x] This PR does not require tests

### Changelog
- [x] Entry has been added to changelog
- [ ] This PR does not make a user-facing change

### Documentation
- [ ] [Docs](https://github.com/plausible/docs) have been updated
- [x] This change does not need a documentation update

### Dark mode
- [ ] The UI has been tested both in dark and light mode
- [x] This PR does not change the UI

### CODE REVIEW

**Comment by DennisLoska** [comment]
@ukutaht The ARM image here is working on RPI4

**Review by paskal — APPROVED** [review]

**Comment by ruslandoga** [comment]
@FarisZR 👋

There's been a change in how public images are built: https://github.com/plausible/analytics/pull/2152
TL;DR what previously was in `.github/workflows/docker.yml` was moved to `.github/workflows/build-public-images.yml`.

I wonder if you'd be up to rebase against master and target the new [`/.github/workflows/build-public-images.yml`](https://github.com/plausible/analytics/blob/054f7f7dbbc63ade81d40add3574b9389c55577d/.github/workflows/build-public-images.yml)?

Either way, thank you!

**Comment by FarisZR** [comment]
Updated, should i include the ARM build for the Private builds too?

**Comment by ruslandoga** [comment]
Thank you!

> should i include the ARM build for the Private builds too?

No, those are for Plausible.Cloud, and they are amd64 only for now :)

**Comment by ruslandoga** [comment]
@ukutaht could you please take a look? I think it's ready.

**Review by ukutaht — APPROVED** [review]

**Comment by XJDHDR** [comment]
Just wanted to make a note: This PR says: "This change does not need a documentation update". However, `self-hosting.md` under `Requirements` says:

> The server must have a CPU with x86_64 architecture and support for SSE 4.2 instructions.

I think that if ARM64 CPUs are now supported, this line and possibly the rest of that section should be updated to reflect this.

**Comment by paskal** [comment]
I was expecting the [master image](https://hub.docker.com/r/plausible/analytics/tags?page=1&name=master) of `plausible/analytics` on DockerHub to be renewed after that PR merging, but it was not, and it seems like it's renewed manually.

I propose to push the new image with both amd64 and arm64.

**Comment by ukutaht** [comment]
@paskal The master and stable images are not updated on DockerHub anymore. Only tagged releases are.

**Comment by asyba** [comment]
so is Raspberry ARM supported or not?

**Comment by ruslandoga** [comment]
@asyba 👋

Starting from `v1.5` [`plausible/analytics`](https://hub.docker.com/r/plausible/analytics/tags) has arm64 support, so the later Raspberry Pi versions should be supported.
